usaspending_api/agency/v2/views/agency_base.py

```diff
@@ -23,6 +23,7 @@ def toptier_code(self):
     @cached_property
     def fiscal_year(self):
         fiscal_year = str(self.request.query_params.get("fiscal_year", current_fiscal_year()))
+        fiscal_year = str(fiscal_year or current_fiscal_year())
```

Comment:
I don't think this line is needed. It would check for an empty string, but if someone passes an empty string then the exception below will be thrown.

Reply:
fair, fixed